### PR TITLE
chore: rename example files

### PR DESCRIPTION
**Pull request by Justintime50**
Since we are using these examples submoduled on our website, we need them to match the templated naming and location expected so they can show on our docs page. 

This PR renames everything, adds missing snippets, and tweaks some linting.

### CODE REVIEW

**Review by nwithan8 — APPROVED** [review]

File: official/docs/java/current/scan-form/create.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import com.easypost.model.Shipment;
+import com.easypost.model.ScanForm;
+
+import java.util.HashMap;
+
+public class Create {
+    public static void main(String[] args) throws EasyPostException {
+        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");
+
+        List<Shipment> shipments = new ArrayList<Shipment>();
```
Comment (jchen293) [inline official/docs/java/current/scan-form/create.java:14]: Missing import statements for `List` and `ArrayList`

File: official/docs/java/current/scan-form/create.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        shipments.add(shipment);
+
+        Map<String, Object> paramMap = new HashMap<String, Object>();
```
Comment (jchen293) [inline official/docs/java/current/scan-form/create.java:17]: Either change `Map` to `HashMap` or import `Map` to this class

File: official/docs/java/current/scan-form/list.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import com.easypost.exception.EasyPostException;
+import com.easypost.model.ScanForm;
+
+import java.util.HashMap;
+
+public class List {
+    public static void main(String[] args) throws EasyPostException {
+        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");
+
+        Map<String, Object> indexMap = new HashMap<String, Object>();
```
Comment (jchen293) [inline official/docs/java/current/scan-form/list.java:13]: Something here about `Map`

File: official/docs/php/current/carbon-offset/create.php [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        'parcel' => [
+            'length' => 20.2,
+            'width' => 10.9,
+            'height' => 5,
+            'weight' => 65.9
+        ]
+    ],
+    null,
+    true
+);
```
Comment (jchen293) [inline official/docs/php/current/carbon-offset/create.php:37]: Missing print statement

File: official/docs/php/current/carbon-offset/one-call-buy.php [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+        'parcel' => [
+            'length' => 20.2,
+            'width' => 10.9,
+            'height' => 5,
+            'weight' => 65.9
+        ]
+    ],
+    null,
+    true
+);
```
Comment (jchen293) [inline official/docs/php/current/carbon-offset/one-call-buy.php:39]: Missing print statement

**Review by jchen293 — CHANGES_REQUESTED** [review]

**Pull request merged**